Added watch to glossary (#47067)
kubernetes/website: content/en/docs/concepts/overview/working-with-objects/labels.md

### LIST and WATCH filtering

> LIST and {{< glossary_tooltip text="WATCH" term_id="watch" >}} operations may specify label selectors to filter the sets of objects

Reviewer: ```suggestion
For **list** and **watch** operations, you can specify label selectors to filter the sets of objects
```

Author: ok . I'll remove the glossary tooltip from here as well.

> returned using a query parameter. Both requirements are permitted

Reviewer: ```suggestion
returned; you specify the filter using a query parameter.
(To learn in detail about watches in Kubernetes, read
[efficient detection of changes](/docs/reference/using-api/api-concepts/#efficient-detection-of-changes)).
Both kinds of requirement are permitted
```
